sales_team_security/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Security rules for sales teams
+==============================
+
+This module sets different permissions levels for accessing sales and CRM
+records based on the sales team: customers, sales orders, leads, opportunities,
+phone calls and sales teams.
+
+It also handles the propagation of the sales team from commercial partners to
+the contacts, which standard doesn't make.
+
+Installation
+============
+
+At installation time, this module sets int all the contacts that have the sales
+team empty the sales team of the parent. If you have a lot of contacts, this
+operation can take a while.
+
+Usage
+=====
+
+On the user configuration (Configuration > Users > Users), select in the
+*Sales Team* section the option "See only own team". Then, the documents
+mentioned before will be filtered out to have only those belonging to the
+teams the user belongs to.
+
+This is complementary to the Sales level access, but sometimes can be
+incoherent depending on the combination chosen. If you chose "See Own Leads"
+on *Sales* section, marking on unmarking the sales team check will be
+irrelevant, because the most restricting level, which the sales one, will
+prevail.
```

Comment:
Wouldn't it make sense to put this module's groups in the same hierarchy as the core sales groups? This way, you can only choose one option...

![captura de pantalla de 2016-08-26 17-52-41](https://cloud.githubusercontent.com/assets/973709/18011620/35fcaa82-6bb6-11e6-85e5-a3e1e7b5dc6c.png)